.github/workflows/ci_e2e_workflow.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
+        python-version: [ 3.6, 3.7, 3.8, 3.9, "3.10" ]
```

Comment:
what does it mean when you enclose it with "" .. does it mean its ignore 3.10?

Reply:
Ref: https://github.com/actions/setup-python/issues/249#issuecomment-934299359